### PR TITLE
Move external domains links and references to HTTPS (where possible)

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
#### Description

In the last years many of the websites are moving from HTTP to HTTPS, many of those work now exclusively in HTTPS.

This PR updates the links and references through the Joomla code from HTTP to HTTPS of the following domains (and their subdomains):
- wikipedia.org (works exclusively HTTPS)
- wikimedia.org (works exclusively HTTPS)
- scribd.com (works exclusively HTTPS)
- facebook.com (works exclusively HTTPS)
- youtube.com (works exclusively HTTPS)
- google.com (works exclusively HTTPS)
- twitter.com (works exclusively HTTPS)
- github.com (works exclusively HTTPS)
- ietf.org
- schema.org

Please note that an effort has been made to not touch external vendor code.
#### How to test

Check the "Files changed" in this PR and verify the changes.
#### More info

For checking reference and links in Joomla code with HTTP has used the egrep linux command: `egrep -R 'http:' /path/to/joomla/`.

Suggestions or corrections are welcome.

### CODE REVIEW

File: tests/unit/stubs/database/jos_finder_links.csv [hunk 1]
```diff
@@ -92,22 +92,22 @@
...
...tegory";a:1:{s:19:"Sample Data-Contact";O:7:"JObject":4:{s:10:"�*�_errors";a:0:{}s:5:"title";s:19:"Sample Data-Contact";s:5:"state";i:1;s:6:"access";i:1;}}s:8:"Language";a:1:{s:5:"en-GB";O:7:"JObject":4:{s:10:"�*�_errors";a:0:{}s:5:"title";s:5:"en-GB";s:5:"state";i:1;s:6:"access";i:1;}}s:6:"Region";a:1:{s:5:"State";O:7:"JObject":4:{s:10:"�*�_errors";a:0:{}s:5:"title";s:5:"State";s:5:"state";i:1;s:6:"access";i:1;}}s:7:"Country";a:1:{s:7:"Country";O:7:"JObject":4:{s:10:"�*�_errors";a:0:{}s:5:"title";s:7:"Country";s:5:"state";i:1;s:6:"access";i:1;}}}s:3:"url";s:46:"index.php?option=com_contact&view=contact&id=1";s:5:"route";s:91:"index.php?option=com_contact&view=contact&id=1:name&catid=16:sample-data-contact&Itemid=229";s:5:"title";s:17:"Contact Name Here";s:11:"description";s:36:"Information about or by the contact.";s:9:"published";N;s:5:"state";s:1:"1";s:6:"access";s:1:"1";s:8:"language";s:5:"en-GB";s:18:"publish_start_date";s:19:"0000-00-00 00:00:00";s:16:"publish_end_date";s:19:"0000-00-00 00:00:00";s:10:"start_date";s:19:"2011-01-01 00:00:01";s:8:"end_date";s:19:"0000-00-00 00:00:00";s:10:"list_price";N;s:10:"sale_price";N;s:7:"type_id";i:3;s:15:"defaultLanguage";s:5:"en-GB";}'
```
Comment (mbabker) [inline tests/unit/stubs/database/jos_finder_links.csv]: Since these lines have serialized content, you'll need to fix the length values as well.

**Comment by andrepereiradasilva** [comment]
yes, of course, working on it

**Comment by brianteeman** [comment]
@andrepereiradasilva can you split this into separate PR - one for language files, one for sample data and another one for php files. Its going to be easier to get it tested and merged that way<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8794">issues.joomla.org/joomla-cms/8794</a>.</sub>

**Comment by andrepereiradasilva** [comment]
well i can do that after 3.5.0 release but only if it's tested and merged in a few days.
I say this because there are a lot of files, and we get conflicts very fast which means that i have to redo over and over again.

**Comment by brianteeman** [comment]
That's why I am asking you to split it up. I can accept the language files
quickly.  Right now it's so big no one will look :(

**Comment by brianteeman** [comment]
And do the same for #8791 or even combine them please<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/8794">issues.joomla.org/joomla-cms/8794</a>.</sub>

**Comment by andrepereiradasilva** [comment]
ok .will combine the two and them split: one for languages ini, other for sample data, other for php files.

**Comment by brianteeman** [comment]
Perfect - thanks

On 12 March 2016 at 14:37, andrepereiradasilva <notifications@github.com>
wrote:

> ok .will combine the two and them split: one for languages ini, other for
> sample data, other for php files.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/8794#issuecomment-195751513>.
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by andrepereiradasilva** [comment]
@brianteeman but we are now in language frezze so that change can only be applied in 3.5.1

**Comment by brianteeman** [comment]
yes but if you do it now. then we can merge as soon as 3.5 is released and
we wont have any conflicts

On 12 March 2016 at 15:00, andrepereiradasilva <notifications@github.com>
wrote:

> @brianteeman <https://github.com/brianteeman> but we are now in language
> frezze so that change can only be applied in 3.5.1
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/joomla/joomla-cms/pull/8794#issuecomment-195753478>.
>



-- 
Brian Teeman
Co-founder Joomla! and OpenSourceMatters Inc.
http://brian.teeman.net/

**Comment by andrepereiradasilva** [comment]
closed as comments above, new PR will be made.

**Pull request closed without merging**